container/root/etc/nginx/sites-available/default

```diff
@@ -44,6 +44,15 @@ server {
     fastcgi_pass 127.0.0.1:9000;
   }
 
+  # Allow locally-accessible (only) FPM status page
+  location ~ ^/__status$ {
+    access_log off;
```

Comment:
`allow 127.0.0.1;` restricts to local. I've manually confirmed that it doesn't expose beyond that. Just trying to keep the noise to a minimum - if this is accessed as part of a healthcheck, that's a duplicate access log